### PR TITLE
Add kustomize artifacts for deploying s3 webserver

### PR DESCRIPTION
**Pull request by accorvin**

### CODE REVIEW

**Comment by anishasthana** [comment]
/lgtm

Built artifacts look good

File: s3-webserver/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      name: s3webserver
+      labels:
+        app: s3webserver
+        component: cvp-webserver-s3webserver
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+        - name: s3webserver
+          imagePullPolicy: Always
+          resources:
```
Comment (anishasthana) [inline s3-webserver/base/deploymentconfig.yaml:25]: Only possible hangup for me is that we're not defining a memory limit. But I'm assuming the usage is usually pretty low.

Comment (accorvin) [reply s3-webserver/base/deploymentconfig.yaml:25]: I had the same thought. I was thinking I'd add the resource limits after deploying on ocp4 and seeing what it's using.

Comment (accorvin) [reply s3-webserver/base/deploymentconfig.yaml:25]: Updated to include a 512m limit based on testing in stage.

**Pull request merged**